src/content/guides/requesting-pre-auth.mdx

```diff
@@ -19,65 +19,44 @@ Pre Auth payments are not supported in all cases.
 
 Pre Auth payments go through an orthogonal payment flow compared to Centrapay’s standard payment flow.
 
+> The example below shows the integrator polling for the Payment Request, this is not needed for eCommerce flows as the integrator will be informed when the authorization has completed.
+
 > See also: [Requesting Payment](/guides/requesting-payment).
 
 ```mermaid
 sequenceDiagram
 	autonumber
 
 	participant Patron
-	participant POS
+	participant Integrator
 	participant Centrapay
 
-	note over Patron, POS: Connect with Patron
-	POS->>Centrapay: Create Authorization
+	note over Patron, Integrator: Connect with Patron
+	Integrator->>Centrapay: Create Authorization
 
-	note over POS: Poll for Payment Confirmation
+	note over Integrator: Poll for Payment Confirmation
 
-	note over POS: ✅ Display Successful Authorization
+	note over Integrator: ✅ Display Successful Authorization
 
-	note over POS, Centrapay: ⏱️ Some time later..
+	note over Integrator, Centrapay: ⏱️ Some time later..
 
 	loop 0..Many (Up to Authorized Amount)
-		POS->>Centrapay: Make Confirmations Against Authorized Funds
-		note over POS: ✅ Display Successful Confirmation
```

Comment:
Removed, the confirmation is normally done as a background process. We don't really care if they display the status since the patron will not be present for this.